module/x/gravity/client/rest/rest.go

```diff
@@ -27,40 +32,40 @@ const (
 // RegisterRoutes - Central function to define routes that get registered by the main application
 func RegisterRoutes(cliCtx client.Context, r *mux.Router, storeName string) {
```

Comment:
TODO: Everything inside this function was commented. Gotta check what should replace it